lib/grpc_mock/response.rb

```diff
@@ -16,7 +16,7 @@ def initialize(exception)
                      end
       end
 
-      def evaluate
```

Comment:
You might want to just add argument `evaluate(_request=nil)` for backward compatibility(I understand the method is only used internally, but I have a preference to keep the method name).